Integrate non-conservative forces and stresses in the ASE calculator (#888)
lab-cosmo/metatensor: metatensor-torch/CHANGELOG.md

- `energy_uncertainty`, `non_conservative_forces` and `non_conservative_stress` are now
  possible outputs for atomistic models

> - The ASE calculator can now use `non_conservative_forces` and `non_conservative_stress`
>   thanks to its new `non_conservative` flag

Reviewer: ```suggestion
- The ASE calculator can now use `non_conservative_forces` and `non_conservative_stress`
  outputs, which are selected with the `non_conservative` flag
```

Author: Done
